backend/spec/features/admin/configuration/shipping_methods_spec.rb

```diff
@@ -38,6 +38,21 @@
       expect(current_path).to eql(spree.edit_admin_shipping_method_path(Spree::ShippingMethod.last))
     end
 
+    it "disables the button at submit", :js do
+      click_link "New Shipping Method"
+      page.execute_script "$('form').submit(function(e) { e.preventDefault()})"
+      fill_in "shipping_method_name", with: "bullock cart"
+
+      within("#shipping_method_categories_field") do
+        check first("input[type='checkbox']")["name"]
+      end
+
+      click_on "Create"
+
+      expect(find('input[type="submit"]')).to be_disabled
```

Comment:
Here as well.

Reply:
@tvdeyen I updated the files according to your suggestion. I think they don't read as good as before, so we're loosing something here. But I agree that saving some seconds is a good thing, so why not 👍